src/pages/dho/Plan.vue

```diff
@@ -190,6 +202,7 @@ export default {
 
 <template lang="pug">
 .page-plan(v-if="!loading")
+  DowngradePopUp(:value="downgradePopUp" @activatePlan="activatePlan" @hidePopUp="downgradePopUp = false")
```

Comment:
@evgenibir We have state var to manage diffrent states. I suggested you change downgradePopUp bool to `state: DOWNGRADING` so we follow the state. Additionally you can add computed vars if you want something like `isDowgradePopUpOpen () { return state === 'DOWNGRADING'}` but i think its not necessary

Reply:
@arsenijesavic fixed!

Reply:
awsome!